README.rst

```diff
@@ -1 +1,47 @@
-See the documentation at https://canonical-starter-pack.readthedocs-hosted.com/use-canonical-sphinx-extension/
+Canonical's Sphinx Starter Pack
+===============================
+
+*A pre-configured repository to build and publish documentation with Sphinx.*
+
+Description
+-----------
+
+The Documentation starter pack includes:
+
+* a bundled Sphinx_ theme, configuration, and extensions
+* support for both reStructuredText (reST) and Markdown
+* build checks for links, spelling, and inclusive language
+* customisation support layered above a core configuration
+
+See the full documentation: https://canonical-starter-pack.readthedocs-hosted.com/use-canonical-sphinx-extension/
+
+Structure
+---------
+
+This section outlines the structure of this repository, and some key files.
+
+init.sh
+*******
+
+This script is an entrypoint intended to simplify adoption of this starter pack.
+Simply download the script, and run it locally.
```

Comment:
```suggestion
Download and run this file it to install the starter pack in an arbitrary directory of the repo.
```
To avoid confusion around how to obtain the file, as it's not a release artefact or anything and people might grab the whole repo.